pom.xml

```diff
@@ -115,6 +117,11 @@
                 <artifactId>okta-http-httpclient</artifactId>
                 <version>${otka.commons.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.okta.commons</groupId>
+                <artifactId>okta-http-okhttp</artifactId>
+                <version>${otka.commons.version}</version>
```

Comment:
Why is okhttp added?

Reply:
Will remove

Reply:
will use it with test scope in impl/pom.xml like you recommended.